widget-reference/chart.md

```diff
@@ -8,6 +8,8 @@ description: >-
 
 ![](../.gitbook/assets/chart.gif)
 
+![](../.gitbook/assets/chart_addchart.gif)
```

Comment:
@ukitta555 can you update the existing chart gif instead of adding a new one and make the gif in the same dimensions and zoom as the prev chart gif